tests/recipes/test_eleuther_eval.py

```diff
@@ -22,21 +22,21 @@ class TestEleutherEval:
     @pytest.mark.parametrize(
         "eval_name, expected_acc, bsz",
         [
-            ("truthfulqa_gen", 0.1, 8),
+            ("truthfulqa_gen", 0.1, 4),
```

Comment:
No need to do bsz 8, that's just slow as hell.

Reply:
honestly, if it doesnt impact the test, can we do 1 and reduce test time?